### PR TITLE
Fix POM of beam-sdks-java-core

### PR DESCRIPTION
**Pull request by mosche**
As of https://github.com/apache/beam/pull/24271 the pom of `beam-sdks-java-core` is broken. There's no version specified for `org.json:json` in `BeamModulePlugin`:

```
[WARNING] The POM for org.apache.beam:beam-sdks-java-core:jar:2.45.0-20221214.234815-14 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for org.apache.beam:beam-sdks-java-core:2.45.0-SNAPSHOT
[ERROR] 'dependencies.dependency.version' for org.json:json:jar is missing. @
```

This sets the version to the transitive version resolved through `com.github.erosb:everit-json-schema:1.14.1`.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by mosche** [comment]
R: @pabloem 
cc @aromanenko-dev

**Comment by aromanenko-dev** [comment]
Run GoPortable PreCommit

**Comment by aromanenko-dev** [comment]
Run Java_Pulsar_IO_Direct PreCommit

**Comment by mosche** [comment]
Run Java_Pulsar_IO_Direct PreCommit

**Comment by mosche** [comment]
Run Java_Kafka_IO_Direct PreCommit

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -679,7 +679,7 @@ class BeamModulePlugin implements Plugin<Project> {
         joda_time                                   : "joda-time:joda-time:2.10.10",
         jsonassert                                  : "org.skyscreamer:jsonassert:1.5.0",
         jsr305                                      : "com.google.code.findbugs:jsr305:$jsr305_version",
-        json_org                                    : "org.json:json",     // Try to keep in sync with google_cloud_platform_libraries_bom transitive deps.
+        json_org                                    : "org.json:json:20220320", // Keep in sync with everit-json-schema / google_cloud_platform_libraries_bom transitive deps.
```
Comment (aromanenko-dev) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:682]: Why not version `20220924` as the latest one?

Comment (mosche) [reply buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:682]: That's the transitive version of com.github.erosb:everit-json-schema:1.14.1 on the classpath

**Pull request merged**